src/app/tab1/tab1.page.html

```diff
@@ -9,19 +29,23 @@
 <ion-content>
   <ion-list lines="none">
     <ion-list-header>
-      <ion-label>trending tv shows</ion-label>
+      <ion-label><h1>Trending tv shows</h1></ion-label>
     </ion-list-header>
     <ion-item ion-item *ngFor="let tvshow of tvshows" >
-      <ion-icon slot="start" color="medium" name="book"></ion-icon>
+      <ion-thumbnail class="thumbnail" slot="start">
+        <ion-img class="thumbnail" [src]="'https://image.tmdb.org/t/p/w200/'+tvshow.poster_path"></ion-img>
```

Comment:
```suggestion
        <ion-img class="thumbnail" [src]="'tvshow.full_poster_path"></ion-img>
```
Netter om deze in je logica alvast in het tvshow object te mappen zodat je hier alleen maar het bovenste hoeft te doen